contrib/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ add_subdirectory (murmurhash)
 add_subdirectory (replxx-cmake)
 add_subdirectory (unixodbc-cmake)
 add_subdirectory (nanodbc-cmake)
+add_subdirectory (yaml-cpp-cmake)
```

Comment:
Clickhouse has various builds, one of those is when ClickHouse is built without libraries (i.e. with `ENABLE_LIBRARIES=0`). I mean cmake scripts should check if `ENABLE_LIBRARIES=0` and if so it should build ClickHouse successfully without `yaml-cpp` (and without `YAMLParser` of course). See `cmake/find/nuraft.cmake` for example how to do that.

Reply:
Done